CMakeLists.txt

```diff
@@ -542,6 +542,10 @@ endif()
 set_source(xsimd)
 resolve_dependency(xsimd 10.0.0)
 
+set(stemmer_SOURCE BUNDLED)
+set_source(stemmer)
+resolve_dependency(stemmer)
```

Comment:
This is now a default addition that is required for all builds? Please also add it to CMake/resolve_dependency/README.md

Reply:
done!